apps/reader/serializers.py

```diff
@@ -27,6 +29,9 @@ def __init__(self, *args, **kwargs):
 
     def create(self, data):
         request = self.context['request']
+        if dict(request.data) == {}:
```

Comment:
Might want to expand this in the future in cases where comment is submitted, but not a rating. This is a good check to ensure that a blank rating is not submitted.